docs: migration guide to the `latest` [1/n] (#17034)
Lightning-AI/lightning: docs/source-pytorch/upgrade/migration_guide.rst

Guide how to upgrade to the 2.0 version
#######################################

> IN the following section we will guide use how to update from almost any Lightning version to the 2.0 release.

Reviewer: ```suggestion
The following section will guide you through updating to the 2.0 release.
```
